ghstack/config.py

```diff
@@ -44,15 +44,12 @@ def read_config(*, request_circle_token: bool = False) -> Config:  # noqa: C901
         config_path = "/".join([str(current_dir), ".ghstackrc"])
         if os.path.exists(config_path):
             break
+        else:
+            config_path = None
```

Comment:
I think it would be more logical if you name the `config_path` binding on line 44 `tentative_config_path` and then assign it to `config_path` in line 46 (so you don't have to None it out here.)